src/app/(routes)/(groups)/games/[slug]/_functions/stats.ts

```diff
@@ -133,25 +134,40 @@ export const calculateMost1st = (
     members.set(placing.player.playerName, ++member);
   }
 
-  return members;
+  const data = Array.from(members);
+
+  return data;
 };
 
+type MembersPerPosition = {
+  first: number;
+  second: number;
+  third: number;
+  last: number;
+} & {
+  [K in 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8]?: number;
+};
 /**
- * Computes per player, their second and last place finishes and returns it.
+ * Computes per player, the amount of times they've placed first, second, third, and last. Also includes their placing 1 - 8.
```

Comment:
So, if I understand this correctly, this stores the number of first seconds, thirds, and lasts as well as the number of times they placed 1-8?

Reply:
Correct.